src/main/java/com/splunk/logging/serialization/EventInfoTypeAdapter.java

```diff
@@ -0,0 +1,67 @@
+/*
+ Copyright © 2019 Splunk Inc.
+ SPLUNK CONFIDENTIAL – Use or disclosure of this material in whole or in part
+ without a valid written license from Splunk Inc. is PROHIBITED.
+ */
+package com.splunk.logging.serialization;
+
+import com.google.gson.*;
+import com.splunk.logging.EventBodySerializer;
+import com.splunk.logging.HttpEventCollectorEventInfo;
+
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public class EventInfoTypeAdapter implements JsonSerializer<HttpEventCollectorEventInfo> {
+
+    @Override
+    public JsonElement serialize(HttpEventCollectorEventInfo src, Type typeOfSrc, JsonSerializationContext context) {
+        Map<String, Object> event = new HashMap<>();
+        // TODO: JsonParser constructor is deprecated in favor of static methods in gson 1.8.6,
```

Comment:
Do we need a jira or will this happen by the time we do v2.0.0?

Reply:
This I think we just keep- I'll do it differently in 2.0.